src/browser-frontend/views/browser/window.jsx

```diff
@@ -33,19 +32,20 @@ import Content from './content';
 })
 export default class Window extends PureComponent {
   componentDidMount() {
+    // Some keyboard shortcuts need to be registered at the platform level,
+    // because they clash with OS-level shortcuts (e.g. Cmd+W on macOS
+    // hides windows, but we want it to close tabs instead.). The platform
+    // is tasked with overriding these shortcuts and providing us control.
```

Comment:
It would not! As described in the top comment, I'd also be happy if we're to decide we want all shortcuts to be handled through the platform completely, not just some of them (Cmd+W for example). However, that introduces some developer effort with making all browser runners (including qbrt) support keyboard shortcut notifications. We want to do this anyway in the long run, though.